habitat/core/benchmark.py

```diff
@@ -131,6 +133,8 @@ def local_evaluate(
         agg_metrics: Dict = defaultdict(float)
 
         count_episodes = 0
+
+        pbar = tqdm(total=num_episodes)
         while count_episodes < num_episodes:
```

Comment:
how about 
```
for count_episodes in tqdm(range(num_episodes)):
```
?

Reply:
I agree this is better, but this code touches all the other challenges as well, so I am worried about consequences of this decision that I don't understand.